src/test/java/org/springframework/data/elasticsearch/repository/support/SimpleReactiveElasticsearchRepositoryTests.java

```diff
@@ -107,7 +109,7 @@ public void saveShouldSaveSingleEntity() {
 	public void saveShouldComputeMultipleEntities() {
 
 		repository
-				.saveAll(Arrays.asList(SampleEntity.builder().build(), SampleEntity.builder().build(),
+				.saveAll(asList(SampleEntity.builder().build(), SampleEntity.builder().build(),
```

Comment:
Nit: We generally don't import `Arrays` statically to avoid clashes.

Reply:
Hmm, I only knew the rule that static imports are ok in tests. Will undo

Reply:
I reverted those changes.